### ISSUE TITLE
Code structure

### ISSUE THREAD

**Issue by epletnev** [fac21/week4-blog-AANR#27]
Very nice organised code with modules

**Comment by RihardsJ** [comment]
I agree but you could improve that since routes folder does not contain only routes:
recomendation -> rename routes to src add 2 subfolders controllers (or handlers) and components. Separate add-blogs, view-blogs, delete-blogs into cotrollers folder and your index.js, blogs-structure into components

**Comment by RihardsJ** [comment]
If you have time you could take your html constants from handlers -> move them into components
